src/cmd4.cc

```diff
@@ -181,7 +181,7 @@ void do_cmd_change_name(void)
 		else if (c == 'f')
 		{
 			strnfmt(tmp, 160, "%s.txt", player_name);
-			if (get_string("Filename(you can post it to http://angband.oook.cz/): ", tmp, 80))
+			if (get_filename("Filename(you can post it to http://angband.oook.cz/): ", tmp, 80))
```

Comment:
(See comment below)

This is an example where "downstream" handles a "~" path. If we could actually start using "boost::filesystem::path" throughout then I'd more optimistic about this approach.